Makefile

```diff
@@ -27,7 +27,7 @@ WEBHOOK_IMAGE_TAG?=$(IMAGE_REPO)/$(APP_NAME)-webhook:latest
 MAIN_PKG=cmd/manager/main.go
 export NAMESPACE?=openshift-sriov-network-operator
 export WATCH_NAMESPACE?=openshift-sriov-network-operator
-export GOFLAGS+=-mod=vendor
+export GOPATH=$(shell go env GOPATH)
```

Comment:
can you change this to:
```
export HOME?=$(PWD)

```

also, can you split this commit that files deletions are separated to other file changes?  It It makes very hard to browse changes.

Reply:
Ah, good idea! 👍🏻 
Sure, let me split it

Reply:
The initial idea why I changed this line in the removal commit was that each commit is working. But I can separate them for sure 👍🏻

Reply:
Yeah, that's a good principle. But in GH it makes reviews pretty impossible :sweat_smile: